annotation-processor/pom.xml

```diff
@@ -21,6 +21,7 @@
 
     <properties>
         <hibernate-validator-parent.path>..</hibernate-validator-parent.path>
+        <automatic.module.name>org.hibernate.validator.ap</automatic.module.name>
```

Comment:
Let's stick to the artifact name convention: this one would be annotationprocessor.